backend/schemas/user.py

```
@@ -1,8 +1,62 @@
+    is_active: bool
+    profile_picture: Optional[str] = None
+    university_name: Optional[str] = None
+    department: Optional[str] = None
+    fields_of_interest: List[str] = []  # Default empty list
+    profile_completed: bool 
+
+    class Config:
+        from_orm = True  # ✅ Correct for Pydantic v2
+
+    @staticmethod
+    def from_orm(user):  # ✅ Safe conversion for merged table
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `from_orm`

(ANN205)